docs/examples/list-filtering/sqlalchemy/sqlalchemy_example/test_authorization.py

```diff
@@ -0,0 +1,94 @@
+from pathlib import Path
+
+import pytest
```

Comment:
Does this need to be in `requirements.txt`?

Reply:
added